pycode/memilio-epidata/memilio/epidata_test/test_epidata_getCaseData.py

```diff
@@ -915,6 +958,8 @@ def test_no_raw(self, mock_loadCsv):
         moving_average = 0
         split_berlin = False
         rep_date = False
+        start_date = date(2020, 4, 24)
+        end_date = date.today()
```

Comment:
Same as in the other test file: Cody copying should be avoided, maybe that makes sense here too.